docs/howto/bill.md

```diff
@@ -50,23 +56,26 @@ AWS management account. If a future cluster deviates from this, you can tell by
 `aws.billing.paid_by_us` being set true in its `cluster.yaml`.
 ```
 
-1. Login to shared AWS SSO at https://2i2c.awsapps.com/start#/
-2. Select the `2i2c-sandbox` account, as it is the primary billing account
-3. Select 'AdministratorAccess' to open the AWS Console for this account
-4. Visit the "Monthly Costs By Linked Account" report ([direct link]) via "Billing and Cost Management" -> "Cost Explorer Saved Reports"
-5. On the right sidebar under "Time -> Date Range", select all the completed months we want to get data for
-6. On the right sidebar under "Time -> Granularity", ensure its selected as "Monthly"
-7. On the right sidebar under "Group by -> Dimension", select "Linked account"
-8. Click the 'Download as CSV' button
-9. Copy AWS costs
-
-   The CSV file has rows for each month, and columns for each project. Copy it
-   into the spreadsheet, making sure the rows and columns both match what is
-   already present. The AWS account name should match the "Cloud Project" row in
-   the spreadsheet.
+1. Login to shared AWS SSO at <https://2i2c.awsapps.com/start#/>
+1. Select the `2i2c-sandbox` account, as it is the primary billing account
+1. Select 'AdministratorAccess' to open the AWS Console for this account
+1. Visit the "Monthly Costs By Linked Account" report ([direct link]) via "Billing and Cost Management" -> "Cost Explorer Saved Reports"
+1. On the right sidebar under "Time -> Date Range", select all the completed months we want to get data for
+1. On the right sidebar under "Time -> Granularity", ensure its selected as "Monthly"
+1. On the right sidebar under "Group by -> Dimension", select "Linked account"
+1. Click the 'Download as CSV' button
+   ```{figure} ../images/aws-billing-ui.jpg
+   AWS billing UI
+   ```
+1. Run the following deployer command to convert the generated CSV file into the format required for the invoicing process.
+   
+   ```bash
+   deployer transform cost-table gcp pathto/downloaded/csvfile
+   ```
 
-   Once done, re-check the numbers once again to make sure they were copied
-   correctly.
+   This will output a new CSV file to your local filesystem called `2i2c_dedicated_cluster_billing_AWS_{START_MONTH}_{END_MONTH}.csv`.
+1. Upload this CSV file to the [cloud costs folder]
+1. Ping the folks in the `#billing` slack channel to let them know the info for dedicated clusters is now available
```

Comment:
This ping should include a link to the file that was just uploaded.

Reply:
Resolved in [`0cdf527` (#4063)](https://github.com/2i2c-org/infrastructure/pull/4063/commits/0cdf5275e7666aa596f777b10bb73e040292e458)